demo/src/ChatContainer.vue

```diff
@@ -46,6 +46,7 @@
 			:menu-actions="JSON.stringify(menuActions)"
 			:message-selection-actions="JSON.stringify(messageSelectionActions)"
 			:templates-text="JSON.stringify(templatesText)"
+			:no-rooms-found-message="'No conversation found'"
```

Comment:
Eu acho que colocar o prefixo `no` fica um pouco ruim de ler. `rooms-not-found-message` não ficaria melhor?

Reply:
Eu também gosto mais do que o @marconiryan falou, começar por `rooms-*` deixa claro que é de room (fácil de buscar no auto-complete).

Reply:
Sim, bem melhor, ajustando...